nasl/nasl_ssh.c

```diff
@@ -733,8 +733,8 @@ nasl_ssh_set_login (lex_ctxt *lexic)
           ssh_options_set (session, SSH_OPTIONS_USER, username))
         {
           g_message ("%s: Failed to set SSH username '%s': %s",
```

Comment:
Also use the style ``Function %s called from %s:`` ?

Similar valid for all other SSH functions